ibm_catalog.json

```diff
@@ -0,0 +1,145 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-ocp-ai",
+        "label": "[Add-ons Beta] Cloud automation for Red Hat OpenShift AI",
+        "product_kind": "solution",
+        "tags": [
+          "containers",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "support_ibm",
+          "ibm_created",
+          "ibm_beta"
+        ],
+        "keywords": [
+          "openshift",
+          "openshift-ai",
+          "ocp-ai",
+          "cluster",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "machine learning",
+          "deep learning",
+          "AI",
+          "artificial intelligence"
+        ],
+        "short_description": "Rapidly deploy Red Hat OpenShift clusters pre-configured with AI capabilities for accelerated machine learning workloads.",
+        "long_description": "This automation streamlines the deployment of Red Hat OpenShift cluster with Artifical Intelligence (AI) addon on IBM Cloud. It accelerates the setup of machine learning environments, reducing the complexity of AI infrastructure management and allowing data scientists and developers to focus on model development and deployment. This offering depends on 'Cloud automation for Red Hat OpenShift VPC' to create a base OCP cluster. Please refer to the released version here (link to OCP DA release). Refer [here](https://cloud.ibm.com/docs/containers?topic=containers-supported-cluster-addon-versions#openshift-ai-416) for supported AI addon versions.",
```

Comment:
I think we can delete this line - 
```
This offering depends on 'Cloud automation for Red Hat OpenShift VPC' to create a base OCP cluster. Please refer to the released version here (link to OCP DA release).
```
Add this line -
```
This is an experimental tile and not suitable for production workloads. Stay here if you want to try an experimental version with the [Optional and swappable components](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-choose-plan-process#optional-swappable) capability.
```

Reply:
Done